code/style.css

```diff
@@ -1,4 +1,208 @@
-html {
-  background: #0025ff;
-  color: #fff;
+*, *::before, *::after {box-sizing: border-box;}
+
+body {
+  /***background: #a3a37d;***/
+  color: #727768;
+  font-family:'Open Sans Condensed', sans-serif;
+  margin: 0px;
 }
+
+/* HEADER+NAVBAR*/
+
+header {
+  background: white;
+  height: 100px;
+}
+
+header:hover {
+  background:#e5e6b4;
+  opacity: 0.5;
+  transition: all 500ms ease;
+  
+}
+
+nav a  {
+  text-decoration: none;
+  color: #727768;
+  margin: 10px;
+  padding: 10px 10px;
```

Comment:
if both vertical and horizontal values are the same (in your case `10px`), you can just type `padding: 10px`